parm/snow/obs/config/bufr_sfcsno_mapping.yaml

```diff
@@ -65,11 +62,18 @@ encoder:
       coordinates: "longitude latitude"
       source: variables/stationIdentification
       longName: "Identification of Observing Location"
-      units: "m"
+      units: "index"
 
     # ObsValue
     - name: "ObsValue/totalSnowDepth"
       coordinates: "longitude latitude"
       source: variables/totalSnowDepth
       longName: "Total Snow Depth"
       units: "mm"
```

Comment:
@jiaruidong2017 The unit of the `totalSnowDepth` defined in IODA data convention is `m`, not `mm`

https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/conventions/tbls.html

Reply:
We should keep to use the `mm` here in the observation files to match the unit used in the UFS model for the snow depth.

Reply:
@jiaruidong2017 Is it possible to read in the totalSnowDepth in `m` unit from the IODA/NetCDF file and then convert it to `mm` in the UFS model?

Reply:
It is possible, but will require additional efforts. We can discuss this issue with the physics team.

Reply:
@jiaruidong2017  I guess only a one-line change is necessary for UFS. You just need to convert `totalSnowDepth` from `m` to `mm` where it is read from IODA obs file.

Reply:
This issue (mismatch between units proscribed by JEDI and those used in the UFS) is bigger than this PR - as all other snow depth IODA files are in mm too. If we change these observations to mm, we'll need to change the others. I suggest that we leave this in mm for now, and create a separate issue to address the unit mismatch (by introducing a unit transform somewhere).

Reply:
Agreed. Let's keep it as "mm" for now.